app/src/main/java/org/mozilla/fenix/experiments/Experiments.kt

```diff
@@ -18,5 +19,6 @@ class ExperimentBranch {
         const val CONTROL = "control"
         const val A1 = "a1"
         const val A2 = "a2"
+        const val BRANCH1 = "branch1"
```

Comment:
You can reuse the `control` and `treatment` if you like.  Those are the default branch names in Experimenter, also.  Unless you have more than 2 branches in your experiment, this would be preferable, or at least a more descriptive branch name maybe?

Reply:
Just to add to this, I would like to standardize on things here if possible so reusing branch names `treatment` and `control` for an A/B experiment like this is defintely my preferred option, and shouldn't cause any issues since they are being used with a different feature-id.

Reply:
That make sense but I added a new branch name as it was part of the ticket description see https://github.com/mozilla-mobile/fenix/issues/18608#issuecomment-813439882 maybe @amedyne or @ItsDevyani  could help us to clarify.

Reply:
I would follow what the Nimbus team prefers. For this experiment, there will be 3 branches. Do you have a recommendation for a standard on how to do this @travis79 ?

Reply:
> I would follow what the Nimbus team prefers. For this experiment, there will be 3 branches. Do you have a recommendation for a standard on how to do this @travis79 ?

In the case where you have more than a control and a treatment branch, then I would say that having them descriptively named is helpful.  I would defer the question to @tdsmith for further guidance on this to get a DS perspective, and maybe he's fine with `branch1` and `branch2`.

Reply:
Also, I only saw 2 branch options, "branch1" and anything else.  Where is the 3rd branch?

Reply:
This evolved from initially 3 different experiments to then being combined to 1 experiment, see #18796 .

Reply:
Hm; as an analyst I'd generally prefer descriptive names to generic names once we have more than two branches. The branch names currently show up in reporting, though we've already identified that that's not ideal. I realize descriptive names are annoying in the context of having to define new constants for each experiment, which isn't something other platforms have required, so it's sort of a novel question for me as well.

I'd have a weak preference for e.g. `settings_menu, `toolbar_menu`, `newtab_card` but if that's a non-starter then we can ride with branch[123].

Reply:
I think adding a prefix to indicate the associated experiment would also be appropriate here. Something like `default_browser_settings_menu` `default_browser_toolbar_menu`. Alternatively, a comment to indicate the constants are branch values for `DEFAULT_BROWSER_TOOLBAR_MENU`.

Reply:
I updated the name to be `DEFAULT_BROWSER_TOOLBAR_MENU`,  please let me know if that it's OK.

Reply:
Works for me, thanks @Amejia481!